src/app/(app)/home/SearchFilterContainer.jsx

```diff
@@ -0,0 +1,14 @@
+import styles from './home.module.scss';
+import { IoSearch } from 'react-icons/io5';
+
+export default function SearchFilterContainer() {
+  return (
+    <div className={styles.searchFilterContainer}>
+      <div className={styles.searchContainer}>
+        <IoSearch />
+        <input type='text' className={styles.searchbar} />
+      </div>
+      <button className={styles.jumpToDateBtn}>Jump to date</button>
+    </div>
+  );
+}
```

Comment:
Not a blocker, but if we are going to deploy a hybrid client/server approach for this feature, Id be interested in deliberating on our file structure strategy. On the one hand having separate files helps with grokkability but on the other it really clutters up the work directories.